include/zephyr/drivers/video.h

```diff
@@ -756,6 +818,69 @@ static inline int video_set_signal(const struct device *dev, struct k_poll_signa
 	return api->set_signal(dev, sig);
 }
 
+/**
+ * @brief Set video selection (crop/compose).
+ *
+ * Configure the optional crop and compose feature of a video device.
+ * Crop is first applied on the input frame, and the result of that crop is applied
+ * to the compose. The result of the compose (width/height) is equal to the format
+ * width/height given to the @ref video_set_format function.
+ *
+ * @param dev Pointer to the device structure for the driver instance.
+ * @param sel Pointer to a video selection structure
+ *
```

Comment:
[non-blocking]
```suggestion
 * Some targets are inter-dependents. For instance, setting a @ref VIDEO_SEL_TGT_CROP will
 * reset @ref VIDEO_SEL_TGT_COMPOSE to the same size when both are supported.
```
Here is some minimum hint for users who can't figure out why their firmware work only when calling functions in different order leads to different results.